### PR TITLE
Added ReverseGeocoding for Nominatim

### PR DESCRIPTION
**Pull request by boldtrn**
Added a simple implementation of Nominatim Reverse Geocoding. OCD is not included here. I was not sure if we should only provide the base nominatim url `http://nominatim.openstreetmap.org/` in the converter.yml.

### CODE REVIEW

File: converter.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
+#TODO it might be better to only specify the base url like http://nominatim.openstreetmap.org/?
```
Comment (karussell) [inline converter.yml]: This is okay for me.

File: src/main/java/com/graphhopper/converter/resources/ConverterResourceNominatim.java [hunk 2]
```diff
@@ -69,8 +79,42 @@ public Response handle(@NotNull @QueryParam("q") String query,
                 get();
 
         Status status = new Status(response.getStatus(), response.getStatusInfo().getReasonPhrase());
-        List<NominatimEntry> entitiesFromResponse = response.readEntity(new GenericType<List<NominatimEntry>>() {
-        });
+        List<NominatimEntry> entitiesFromResponse;
+        if (reverse) {
+            System.out.println(response);
```
Comment (karussell) [inline src/main/java/com/graphhopper/converter/resources/ConverterResourceNominatim.java]: This should be commented out :)

**Comment by karussell** [comment]
Thanks - looks good (just the minor comment)

**Comment by boldtrn** [comment]
@karussell Thanks, I merged my changes.

**Comment by karussell** [comment]
Thanks!

**Pull request merged**